internal/backend/db/dbgorm/sessions.go

```diff
@@ -353,6 +353,7 @@ func (gdb *gormdb) getSessionCallAttemptResult(ctx context.Context, sessionID sd
 }
 
 func (db *gormdb) CreateSession(ctx context.Context, session sdktypes.Session) error {
+	// REVIEW: pass pointer and update createdAT and updatedAt?
```

Comment:
?

Reply:
we update session.createdAt only in service. but since its passed by value you cannot benefit from it.
So I'm thinking maybe we could pass by ref here and update time

Reply:
let's raise with everyone
not sure what's the correct answer here

Reply:
rewritten differently. we cannot pass by reference and change sdktype anyway